data/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This file should ideally go in the project's root directory. It *could* go in the server directory, and it would work, but by putting it in the root directory, you could, in the future, add additional dependencies that would exist outside the server directory. By placing it in the data directory as it currently is, and consequently the node_modules directory as well, the underscore dependency isn't resolving in your trucks module.